Cargo.toml

```diff
@@ -20,3 +20,8 @@ lto = true
 [profile.release-with-debug]
 inherits = "release"
 debug = true
+
+[workspace.dependencies]
```

Comment:
Huh that's neat, didnt know you can do that

Reply:
Yeah just found out about it too, keeps stuff a bit more organized